code/campaign/src/main/java/com/adobe/marketing/mobile/campaign/LocalNotificationService.java

```diff
@@ -0,0 +1,110 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+ */
+package com.adobe.marketing.mobile.campaign;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+import com.adobe.marketing.mobile.services.Log;
+import com.adobe.marketing.mobile.util.MapUtils;
+
+import java.security.SecureRandom;
+import java.util.Calendar;
+import java.util.HashMap;
+
+class LocalNotificationService {
+    private static final String LOG_TAG = "Campaign";
+    private static final String SELF_TAG = "LocalNotificationService";
+    public static final String NOTIFICATION_CONTENT_KEY = "NOTIFICATION_CONTENT";
```

Comment:
do these static strings need to be public?

Reply:
Changed to private